examples/experiments/keyrelease_demo.py

```diff
@@ -0,0 +1,53 @@
+"""
+=============
+KeyPressAndRelease demo
+=============
+
+This example demonstrates gathering key-releases as well as presses with
+the ExperimentController class.
```

Comment:
Please add a `.. warning::` directive noting that it currently does not work on TDT or Cedrus boxes, only keyboard. Also please add a similar warning in the `get_presses` docstring. Since we don't guarantee accurate timing for anything other than TDT and Cedrus use, it's important that people know the timing isn't tested, and is probably bad, when using keyboard and/or Pyglet audio.